{{cookiecutter.service_name}}/service.py

```diff
@@ -510,7 +512,13 @@ def {{cookiecutter.workflow_id |replace("-", "_")  }}(conf, inputs, outputs): #
 
     except Exception as e:
         logger.error("ERROR in processing execution template...")
+        logger.error("Try fetching logs if any...")
```

Comment:
I think this should become logger.info as it is not indicative of an error itself

Reply:
updated